pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[project]
+name = "gon"
+description = "Processing of planar geometries."
+authors = [{ name = "Azat Ibrakov", email = "azatibrakov@gmail.com" }]
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy"
+]
+requires-python = ">=3.7"
+dependencies = [
+    "bentley-ottmann>=8.0.0,<9.0",
+    "clipping>=6.0.0,<7.0",
+    "ground>=9.0.0,<10.0",
+    "locus>=10.0.0,<11.0",
+    "orient>=7.0.0,<8.0",
+    "reprit>=0.9.0,<1.0",
+    "sect>=7.0.0,<8.0",
+    "symba>=2.2.1,<3.0"
```

Comment:
Consider using this syntax instead: https://python-poetry.org/docs/dependency-specification/#caret-requirements

Reply:
we are not using `poetry`